core/templates/dev/head/base.js

```diff
@@ -211,6 +213,13 @@ oppia.controller('Base', [
       return false;
     };
 
+    //  Method to check if the window size is narrow
+    $scope.windowSize = function() {
+      $scope.isWindowNarrow = windowDimensionsService.getWidth() <= 768;
```

Comment:
Don't you need to set $scope.isWindowNarrow at the outset, as well? Or does the resize hook also trigger when the page is loaded?